framework/source/class/qx/ui/layout/Grid.js

```diff
@@ -93,8 +99,10 @@ qx.Class.define("qx.ui.layout.Grid",
    *     Sets {@link #spacingX}.
    * @param spacingY {Integer?0} The vertical spacing between grid cells.
    *     Sets {@link #spacingY}.
+   * @param allowGrowSpannedCellWidth {Boolean?false} Allow growing of spanning cell widths beyond the cumulated widths of the columns.
+   *     Sets {@link #allowGrowSpannedCellWidth}.
    */
-  construct : function(spacingX, spacingY)
+  construct : function(spacingX, spacingY, allowGrowSpannedCellWidth)
```

Comment:
hmmm I am  bit ambivalent about adding extra positional arguments  ...

Reply:
I thought that this is suitable because it is a global acting layout property, just as spacingX and spacingY are.

Reply:
I think I agree with @oetiker that it's not necessary there, and it doesn't really match other classes which also have spacing parameters but not anything like this. It's easy enough to set the property if something other than the default is desired.

Reply:
@derrell removed the additional parameter